config/routes.rb

```diff
@@ -3,23 +3,28 @@
     sessions: 'sessions/sessions'
   }
 
-  resources :clients
-  resources :intakes
-  resources :vouchers
-  get "vouchers/:id/created" => "vouchers#created", as: :voucher_created
-  resources :swaps
+  constraints(lambda { |req| req.env["warden"].user(:user).intake_user? }) do
```

Comment:
Constraints, without a namespace. Since our "main" user is the intake user, it would seem silly to have routes namespaced for them like `/intake/intake/new`. So we have routing constraints to respect the flag, but not a namesapce